druid-shell/src/platform/mac/window.rs

```diff
@@ -56,13 +56,44 @@ use crate::keyboard_types::KeyState;
 use crate::mouse::{Cursor, MouseButton, MouseButtons, MouseEvent};
 use crate::region::Region;
 use crate::scale::Scale;
-use crate::window::{IdleToken, TimerToken, WinHandler, WindowState};
+use crate::window::{IdleToken, TimerToken, WinHandler, WindowState, WindowLevel};
 use crate::Error;
 
 
 #[allow(non_upper_case_globals)]
 const NSWindowDidBecomeKeyNotification: &str = "NSWindowDidBecomeKeyNotification";
 
+#[allow( dead_code)]
+#[allow(non_upper_case_globals)]
+mod levels{
```

Comment:
I guess you forgot to `rustfmt`?

Reply:
Nope, it passed rustfmt.

Reply:
Not sure why it didn't fail and sorry for being picky!

These are not [1](https://github.com/linebender/druid/pull/1231/files#diff-19ff16b764821e9edacfc74a3d8830a1R713-R714), [2](https://github.com/linebender/druid/pull/1231/files#diff-1235e00de085eb8a53e79bb902f0afb2R68), [3](https://github.com/linebender/druid/pull/1231/files#diff-1235e00de085eb8a53e79bb902f0afb2R77), [4](https://github.com/linebender/druid/pull/1231/files#diff-1235e00de085eb8a53e79bb902f0afb2R82-R84), [5](https://github.com/linebender/druid/pull/1231/files#diff-1235e00de085eb8a53e79bb902f0afb2R466), [6](https://github.com/linebender/druid/pull/1231/files#diff-1235e00de085eb8a53e79bb902f0afb2R473), [7](https://github.com/linebender/druid/pull/1231/files#diff-1235e00de085eb8a53e79bb902f0afb2R982) correctly formatted?

Reply:
Your links are not working for me. I don’t think we need a human Rustfmt impersonator to be honest

Reply:
Yeah, something about the conditional compilation in the platform-specific code confuses rustfmt. This has come up before...

Reply:
I ended up doing the individual files. A bit annoying